Jenkinsfile

```diff
@@ -0,0 +1,47 @@
+pipeline {
+    agent any
+    stages {
+        stage('SCM') {
+            steps {
+                checkout scm
+            }
+        }
+
+        stage('Download Build Wrapper') {
+            steps {
+                powershell '''
+                  $path = "$HOME/.sonar/build-wrapper-win-x86.zip"
+                  rm build-wrapper-win-x86 -Recurse -Force -ErrorAction SilentlyContinue
```

Comment:
We are removing the dirs, as if they could exits from previous builds, but in the cmake we just call build. Is there a some special meaning for $HOME (so it is not cleaned between jobs), and different from the normal working directory of jenkins?
If so, maybe create .sonar dir not in home, but working directory?

Reply:
You're right, that's a mistake. I'll correct it